plugin/assets/css/src/admin.css

```diff
@@ -84,3 +84,23 @@
 		display: none;
 	}
 }
+
+.material_design_update_message {
+	background-color: #f5f5f5;
+	padding: 1rem;
+	font-weight: bold;
+	display: block;
+	margin-top: 1rem;
+	border: 1px solid #ccd0d4;
+
+	&::before {
```

Comment:
This is sass and not css right?

Reply:
@rodydavis There's a post-css plugin installed that allows us to use sass-like selectors